src/SFA.DAS.LearnerDataMismatches.UnitTests/Web/Infrastructure/EmployerServiceTest.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EAS.Account.Api.Client;
+using SFA.DAS.EAS.Account.Api.Types;
+using SFA.DAS.LearnerDataMismatches.Web.Infrastructure;
+
+namespace SFA.DAS.LearnerDataMismatches.UnitTests.Web.Infrastructure
+{
+    public class EmployerServiceTest
+    {
+        private readonly Mock<IAccountApiClient> _mockAccountApiClient = new Mock<IAccountApiClient>();
+
+        [Test]
+        public async Task WhenInvalidEmployerAccountId_ThenReturnEmptyString()
```

Comment:
I prefer these tests to the previous PR as they are very easy to read without navigating around the class 👍